app/presenters/answer_presenter.rb

```diff
@@ -1,4 +1,23 @@
 class AnswerPresenter < ContentItemPresenter
   include Body
   include LastUpdated
+
+  def ab_body
+    b_button = <<EOD
+      <div class="transaction">
+        <p class="get-started group">
+          <a id="epilepsy-driving-variant-b" class="button" href="https://www.driving-medical-condition.service.gov.uk/eligibility/entitlement-gb">
```

Comment:
When I click the link to https://www.driving-medical-condition.service.gov.uk/eligibility/entitlement-gb I get a "Your session has timed out" error.

Clicking the link takes me here:
![screen shot 2017-05-22 at 08 04 01](https://cloud.githubusercontent.com/assets/319055/26296156/49f34cb6-3ec5-11e7-8fdd-554406ef9fb4.png)

Clicking "Start again" takes me here:
![screen shot 2017-05-22 at 08 04 14](https://cloud.githubusercontent.com/assets/319055/26296163/527f355c-3ec5-11e7-9e7c-ad86bab08c7e.png)